### PR TITLE
Add charm tools for people getting started

### PR DESCRIPTION
**Pull request by VariableDeclared**
Added a mkcharmtree script to create the directory structure for newbies.

### CODE REVIEW

**Comment by VariableDeclared** [comment]
Hey @niemeyer, yes I see you point, honestly I just wanted to draw some attention to this. And if we got the foundations of some tools in there, then good, otherwise we'd have a discussion.

I managed to spend some more time with the framework yesterday, so let me go away and redesign this script so the UX isn't so rubbish.

Cheers for your input.

File: charm_tools/letscharm.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    setup_charm_dir(charm_name)
+    initialise_tree(charm_name)
+
+    k8s_charm = input("Is this a k8s charm? y/n")
+    if k8s_charm.lower() == "y":
+        print("Linking hooks/start")
+        run_command("ln -s ../src/charm.py hooks/start")
+    else:
+        run_command("ln -s ../src/charm.py hooks/install")
+        print("Linking hooks/install")
```
Comment (johnsca) [inline charm_tools/letscharm.py]: These extra outputs seem redundant with the "Running command" output, so I'd suggest removing them both.  However, if you want to keep them, this one should come before the `run_command`

**Comment by facundobatista** [comment]
IMO all the commands issued to the system (maybe the exception being `git`) must be done through the Python stdlib. 

IOW don't do `subprocess.call("mkdir foo")`, do `os.mkdir("foo")`.

**Comment by chipaca** [comment]
to me this feels like something that i'd want to have in a companion tool, not in the operator framework. That is, I'd expect the next-gen `charm` command to do this.

**Comment by chipaca** [comment]
I'm going to close this for now. Yes we want something like this, but not in this project itself. At least not at this point in time. Exactly where this should live is still TBD.

Thank you for pointing out the need to me, though.

**Pull request closed without merging**